decidim-proposals/db/migrate/20240110203500_add_withdrawn_fields_on_proposals.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AddWithdrawnFieldsOnProposals < ActiveRecord::Migration[6.1]
+  def up
+    add_column :decidim_proposals_proposals, :withdrawn_at, :datetime
+    add_column :decidim_proposals_proposals, :withdrawn, :boolean, default: false, null: false
```

Comment:
Can you leave only the withdrawn_at column and remove the withdrawn column? We can add a `withdrawn?` method to make it easier to work with like  https://github.com/decidim/decidim/blob/02811d37f74953dcc461556ecf5eb0fc190a7064/decidim-core/app/models/decidim/user.rb#L144

Reply:
Yes, we can work with this.